README.md

```diff
@@ -408,36 +408,41 @@ will produce a check for a number less than 0.
 
 ##### Transforming Validators
 There are a few built in Validators that let you transform values if used with the `transform` function.  If they also do some checks then they can be used `check` but some may do nothing in that case.
- * 🆕 `CLI::Bounded(min,max)` will bound values between min and max and values outside of that range are limited to min or max,  it will fail if the value cannot be converted and produce a `ValidationError`
```

Comment:
Why are you changing stars to dashes? I like stars *slightly* better than dashes, since they look more like the final result. If there's a reason to change, I'd be happy to change.

Reply:
A couple reasons, neither of which I consider definitive,  the first is consistency, we have used dashes in some spots and `*` in others,  I don't have a strong preference but I use `-` in everything else I edit,  I am fine either way.  The other is I by default run a markdown linter in my editor, and that always shows some warning, so I have slowly been resolving them with the `-`.   

Like I said a few non-definitive reasons if you want them changed back I would be happy to do so,  though some clarity on style might be useful.

Reply:
We can go to all dashes, I'd be okay with that; a linter liking dashes is possibly enough of a reason to change.